fft.py

```diff
@@ -8,40 +9,30 @@
 
 import numpy as np
 
-#Main code for the periodogram
-import matplotlib.pyplot as plt
-
-
 """
 https://www.ritchievink.com/blog/2017/04/23/understanding-the-fourier-transform-by-example/
 """
 
-def DFT(x):
-    """
-    Compute the discrete Fourier Transform of the 1D array x
-    :param x: (array)
-    """
+# def DFT(x):
+#     """
+#     Compute the discrete Fourier Transform of the 1D array x
+#     :param x: (array)
+#     """
 
-    N = x.size
-    n = np.arange(N)
-    k = n.reshape((N, 1))
-    e = np.exp(-2j * np.pi * k * n / N)
-    return np.dot(e, x),N
+#     N = x.size
+#     n = np.arange(N)
+#     k = n.reshape((N, 1))
+#     e = np.exp(-2j * np.pi * k * n / N)
+#     return np.dot(e, x),N
 
-import numpy as np
 def npfft(x,t):
     '''Return the normalised amplitude of fft and frequency using fast FFT in Numpy'''
     import numpy as np
     N=x.size
     fft = np.fft.fft(x)
     T = t[1] - t[0]  # sampling interval 
     # 1/T = frequency
-    f = np.linspace(0, 1 / T, N)
```

Comment:
Use numpy.fft.fftfreq to calculate the frequency.

https://het.as.utexas.edu/HET/Software/Numpy/reference/generated/numpy.fft.fftfreq.html